core/src/main/java/org/wso2/testgrid/core/command/GenerateTestPlanCommand.java

```diff
@@ -403,6 +403,8 @@ private void populateDefaults(TestgridYaml testgridYaml) {
             DeploymentConfig.DeploymentPattern deploymentPatternConfig = new DeploymentConfig.DeploymentPattern();
             deploymentPatternConfig.setName(TestGridConstants.DEFAULT_DEPLOYMENT_PATTERN_NAME);
             deploymentPatternConfig.setDescription(TestGridConstants.DEFAULT_DEPLOYMENT_PATTERN_NAME);
+            deploymentPatternConfig.setRemoteRepository(TestGridConstants.REPO_NOT_CONFIGURED_STR);
+            deploymentPatternConfig.setRemoteBranch(TestGridConstants.REPO_NOT_CONFIGURED_STR);
```

Comment:
Since we use this constant for both Repository and Branch, we can name the constant as `NOT_CONFIGURED_STR` , WDYT?